spellcheck/README

```diff
@@ -28,15 +28,15 @@ Features
 
 Usage
 -----
-After installed successfully, load the plugin in Geany's plugin manager
-and a new menu item in the Tools menu will appear. Alternatively, you can
-assign a keyboard shortcut in Geany's preferences dialog to perform a
-spell check.
+[Install the plugin](https://plugins.geany.org/install.html) then
```

Comment:
This readme isn't markdown, its plain text, so you shouldn't put markdown markup :) in it.  There is a readme.rst with markup, and thats what github renders.

Reply:
I'm sorry, all of GitHub is Markdown :) Feel free to change it, the idea was [this](https://github.com/geany/geany/issues/2508)